README.md

```diff
@@ -1,2 +1,2 @@
 # module-ballerinax-mssql
-Ballerina MSSQL DB module
+Ballerina MSSQL DB module
```

Comment:
Update README & Package.md

Reply:
Fixed with [`fe3790a`](https://github.com/ballerina-platform/module-ballerinax-mssql/pull/2/commits/fe3790ae0589bafee2afeb9e29e8002bc9a88529) and [`bbfddbf`](https://github.com/ballerina-platform/module-ballerinax-mssql/pull/2/commits/bbfddbffb41c7de434010043f01f94b1df2ccc3c)